pom.xml

```diff
@@ -1,156 +1,167 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
non-blocking:
 the diff of this file is hard to read with the reformatting (i presume from your IDE) changing more of the file than is explicit to this PR. it's help a lot if such reformatting changes are kept isolated in separate commits, or if you can disable reformatting in your IDE.